app/helpers/format-date.js

```diff
@@ -1,7 +1,8 @@
 import { helper } from '@ember/component/helper';
 
 export function formatDate(date) {
-  return new Intl.DateTimeFormat('en-GB')
+  // TODO: Use real user locale and sync to language.
+  return new Intl.DateTimeFormat('fr-CA')
```

Comment:
```suggestion
  return new Intl.DateTimeFormat(navigator.language ?? 'en-US')
```

Reply:
yeah, but i am in the us and we have a stupid date format.  hahaha.  I want this to be ISO8601 formatted.

Reply:
I added a second, explicit ISO8601 method here and made the default one use the user locale.